sway/desktop/idle_inhibit_v1.c

```diff
@@ -58,8 +65,9 @@ void sway_idle_inhibit_v1_user_inhibitor_register(struct sway_view *view,
 	inhibitor->view = view;
 	wl_list_insert(&inhibitor->manager->inhibitors, &inhibitor->link);
 
-	inhibitor->destroy.notify = handle_destroy;
-	wl_signal_add(&view->events.unmap, &inhibitor->destroy);
```

Comment:
Hm, why isn't `unmap` emitted before `destroy`?

Reply:
Unmap is likely fine - hadn't dug into the wlroots side yet. However, the section for idle_inhibit only listened on inhibitor destroy, so that's the primary change.